index.html

```diff
@@ -1,411 +1,383 @@
+    </header>
+    <section class="main_container">
+      <div class="intro">
+        <div class="intro_item intro_item1">
+          <h1>Hey There. I'm Farida</h1>
+          <h2>I'am a Software Developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have
+            a project you need coded, don't hestiate to contact me.
+          </p>
+        </div>
+        <div class="intro_item intro_item2">
+          <div class="social_media">
+            <ul class="social_media_head">
+              <li>
+                <a href="#">
+                  <img class="img img1" src="img/github.png" alt="GitHub" /></a>
+              </li>
+              <li>
+                <a href="#"><img class="img img2" src="img/linkin.png" alt="Linkedin" /></a>
+              </li>
+              <li>
+                <a href="#"><img class="img img3" src="img/yo.png" alt="Angelist" /></a>
+              </li>
+              <li>
+                <a href="#"><img class="img img4" src="img/twitter.png" alt="Twitter" /></a>
+              </li>
+              <li>
+                <a href="#"><img class="img img5" src="img/Vector.png" alt="Vector" /></a>
+              </li>
+            </ul>
           </div>
         </div>
-      </section>
-      <section class="works">
+        <div class="intro_item intro_item3" id="intro_image">
+          <img src="img/Illustration-header2x.png" width="100%" height="100%" alt="Header image">
+        </div>
+      </div>
+    </section>
+    <section class="works">
+      <div class="work_head">
         <div class="works_heading">
           <h3>My Recent Works</h3>
         </div>
-        <hr class="underline" />
-        <div class="recent_works">
-          <div>
-            <img width="100%" src="img/works/imgPlaceholder.png" alt="Work" />
+        <div class="underline">
+          <!-- <hr class="underline" /> -->
+        </div>
+      </div>
+
+      <div class="recent_works">
+        <div class="recent_works_img">
+          <img width="80%" src="img/works/imgPlaceholder.png" alt="Work" />
+        </div>
+        <div class="multi_post">
+          <h3>Multi-Post Stories</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a standard
+            dummy text.
+          </p>
+          <ul class="stacks">
+            <li class="stack_item stack_item1">css</li>
+            <li class="stack_item stack_item2">html</li>
+            <li class="stack_item stack_item3">bootstrap</li>
+            <li class="stack_item stack_item4">Ruby</li>
+          </ul>
+          <button type="button" class="see_projects_btn btn_text">
+            <span class="btn_text">See Project</span>
+          </button>
```

Comment:
- [ ] Great job with your design 👏. Kindly make sure there is no `button` in your design until one of the card is hovered 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206718440-605eab0c-8eb2-4782-923b-59ac5fbd401d.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206718403-dc0ee458-76ff-42de-943f-85484357e391.png)

- [ ] Kindly make sure when one of the cards are hovered on the `linear-gradient` is removed and your design looks as close as possible to your chosen Figma design 🙏 

![image](https://user-images.githubusercontent.com/74814780/206719014-c4a77fa6-b213-426c-afa8-99d8e86ea96a.png)